Lemnisk latest doc (#1136)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/lemnisk/index.mdx

---

> title: "Lemnisk"

Reviewer: Can we convert it into Lemnisk Marketing Automation?

description: Send your event data from RudderStack to Lemnisk.
aliases: ["Lemnisk Marketing Automation", "Lemnisk CDP"]
---

> [Lemnisk](https://www.lemnisk.co/) is a Customer Data Platform which provides personalization and real-time marketing automation solution. It aims to deliver superior customer experiences that result in increased conversions, retention, and growth for enterprises.

Reviewer 2: ```suggestion
[Lemnisk](https://www.lemnisk.co/) is a Customer Data Platform that provides personalization and real-time marketing automation solutions. It delivers superior customer experiences that increase user conversions, retention, and growth.
```
